client/src/Q&As/Modal.jsx

```diff
@@ -37,24 +31,31 @@ const Modal = ({origin, title, subTitle, questionBody, onClose, show}) => {
     }
   };
 
+  useEffect(() => {
+    if (isItAnswer) {
```

Comment:
Good idea to set the modal like this to cut down repeptive code

Reply:
😀